source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/deepks_lcao.cpp

```diff
@@ -347,25 +347,46 @@ void hamilt::DeePKS<hamilt::OperatorLCAO<TK, TR>>::calculate_HR()
         std::vector<int> trace_alpha_row;
         std::vector<int> trace_alpha_col;
         std::vector<double> gedms;
-        int ib=0;
-        for (int L0 = 0; L0 <= orb.Alpha[0].getLmax();++L0)
+        if(!GlobalC::ld.get_if_equiv())
```

Comment:
Do not use GlobalC.

Reply:
it would be possible not using globalc::ld if the calculation of V_delta has not been moved out of the deepks class. Unfortunately this is not an option now as a result of previous refactoring by someone else.